samples/keys/hw_unique_key/README.rst

```diff
@@ -0,0 +1,96 @@
+.. _hw_unique_key_usage:
+
+Hardware unique key
+###################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample shows how to use a hardware unique key (HUK) to derive an encryption key through psa_crypto APIs.
+
+Overview
+********
+
+The sample goes through the following steps to derive a key and use it to encrypt a string:
+
+1. Initializing the hardware, if applicable.
+
+   When TF-M is enabled, the hardware is initialized automatically when booting.
+
+#. Generating HUKs, if applicable.
+
+   When TF-M is enabled, the HUKs are generated automatically when booting.
```

Comment:
Should we explain that you need to reboot to get the key to cryptocell if `CONFIG_SECURE_BOOT` is set instead of TF-M ?

Reply:
Added some text about this